tests/drivers/uart/uart_elementary/boards/nrf5340dk_nrf5340_cpuapp.overlay

```diff
@@ -0,0 +1,40 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+
+&pinctrl {
+	uart1_default_alt: uart1_default_alt {
+		group1 {
+			psels = <NRF_PSEL(UART_TX, 0, 4)>,
+				<NRF_PSEL(UART_RX, 0, 5)>,
+				<NRF_PSEL(UART_RTS, 0, 6)>,
+				<NRF_PSEL(UART_CTS, 0, 7)>;
+		};
+	};
+
+	uart1_sleep_alt: uart1_sleep_alt {
+		group1 {
+			psels = <NRF_PSEL(UART_TX, 0, 4)>,
+				<NRF_PSEL(UART_RX, 0, 5)>,
+				<NRF_PSEL(UART_RTS, 0, 6)>,
+				<NRF_PSEL(UART_CTS, 0, 7)>;
+			low-power-enable;
+		};
+	};
+};
+
+dut: &uart1 {
+	current-speed = <115200>;
+	compatible = "nordic,nrf-uarte";
+	status = "okay";
+	pinctrl-0 = <&uart1_default_alt>;
+	pinctrl-1 = <&uart1_sleep_alt>;
+	pinctrl-names = "default", "sleep";
+	hw-flow-control;
+};
+
+&rtc1 {
+	interrupts = <21 2>;
+};
+
+counter_dev: &timer0 {
```

Comment:
can be removed. Looks like copy-paste from uart_mix_fifo_poll test where it was used.